pyfr/writers/paraview.py

```diff
@@ -63,6 +65,75 @@ def write_out(self):
         write_s('\n</AppendedData>\n</VTKFile>')
 
 
+class ParallelParaviewWriter(BaseWriter):
+    """Wrapper for writing serial .vtu Paraview files"""
+    # Supported file types and extensions
+    name = 'paraview'
```

Comment:
Name must be unique across subclasses (I'll comment on a better way to handle this later).